### DISCUSSION TITLE
Motors running with the drone turned off and completely inaccessible to the client

### DISCUSSION THREAD

**Discussion by otaviozordan** [bitcraze/discussions#1560] [Q&A]
Hello everybody,
I'm a student and received this drone to work on stability and control. This is my first time using it. 
It's a Crazyflie 2.1.

The problems start following the Get Start, the drone came to me with a Flow V2 board below and when connecting the drone to the USB it seems to initialize incorrectly. Then I removed the Flow V2 board and verified that initialization seemed to occur correctly. 
After that I tried connecting it to the battery, another surprise, when connecting the propellers on the back of the drone they started to spin instantly and worse, when turning off the drone they started to spin faster! While the drone is connected to the battery it becomes incommunicable via radio. I desperately need help

I already reinstalled the firmware and connected to it with the drone powered by usb only. If I connect it to the battery it becomes inaccessible

https://github.com/user-attachments/assets/1dad1a3d-f8d0-43c3-ae18-b843ec810dda

**Comment by knmcguire** [comment]
Hi! This seems to be a hardware issue. Could you please send an email to contact _ at_ bitcraze.io to ask for a replacement?
